save-common/src/commonMain/kotlin/org/cqfn/save/core/plugin/Plugin.kt

```diff
@@ -84,9 +91,24 @@ abstract class Plugin(
             }
 
         return if (testFiles.isNotEmpty()) {
-            rawTestFiles.filter { rawTestFile ->
-                testFiles.any { it in rawTestFile.test.toString() }
+            val (foundTest, notFoundTest) = rawTestFiles.partition { rawTestFile ->
+                testFiles.any {
+                    try {
+                        if (fs.metadata(it.toPath()).isDirectory) {
```

Comment:
Find out, that https://github.com/cqfn/save/issues/235 is actual only when `testRootPath == .`, since we connecting the paths (`testRootPath` and path to the test file) in 
https://github.com/cqfn/save/blob/fd397c15412b918a36a01e3271dadff62236bd7a/save-core/src/commonNonJsMain/kotlin/org/cqfn/save/core/Save.kt#L60

Thus, only in case `testRootPath == .` we will obtain incorrect behavior. Result of concatenation will contain only the name of test file.

In other cases, the concatenation 
from `Save.kt` will have incorrect value, which won't pass the condition of
`it in rawTestFile.test.toString()`

So, I think we can simplify the logic and just check, whether the `it.toPath()` exists. And no need to check, whether it is a directory and let the remaining logic as it were before.